src/pages/Home/index.tsx

```diff
@@ -64,8 +66,27 @@ const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
     }
   }, [searchQuery]);
 
-  // console.log(order, 'this is order3'); // this is for you, u said you want to see the console.log
+  useEffect(() => {
+    fetch('http://127.0.0.1:8000/boba_list')
+      .then((response) => {
+        if (response.ok) {
+          return response.json();
+        }
+      })
```

Comment:
Can you also try testing the page without the backend server running?
Any error? The site shouldn't crash with nasty error so display something like 503

Reply:
like you want me to redirect to like error page( which we dont have it yet ) or ( just <div> 503 server unavaliable <div>

Reply:
You don't need to redirect.
If the server status return as not available.
Just render the component with some kind of bootstrap component in a nice way.

Reply:
Add anything like this without creating any redirect or route.
Since we don't handle react router yet. 

https://dribbble.com/tags/503

Reply:
fixed at eb54ecad74f59dbb68a5e237a50d416a18e76b71